Core/Core/Modules/ModuleItems/ModuleItemSequenceViewController.swift

```diff
@@ -58,28 +58,36 @@ public class ModuleItemSequenceViewController: UIViewController {
         pages.scrollView.isScrollEnabled = false
         embed(pages, in: pagesContainer)
 
+        // places the next arrow on the opposite side
+        let transform = CGAffineTransform(scaleX: -1, y: 1)
+        nextButton.transform = transform
+        nextButton.titleLabel?.transform = transform
+        nextButton.imageView?.transform = transform
```

Comment:
interesting. I think I like this better than the forcing the RTL semantic on the button to put the image on the other side.